x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,352 @@
+                  defaultMessage="Tags"
+                />
+              </EuiDescriptionListTitle>
+              <EuiSpacer size="s" />
+
+              {module.tags?.length ? (
+                module.tags.map((tag) => <ListDescriptionItem value={tag} />)
+              ) : (
+                <ListDescriptionItem
+                  // @ts-ignore
+                  value={
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.noTagsLabel"
+                      defaultMessage="No tags"
+                    />
+                  }
+                />
+              )}
+
+              <EuiSpacer size="m" />
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.typeTitle"
+                  defaultMessage="Type"
+                />
+                <EuiSpacer size="s" />
+              </EuiDescriptionListTitle>
+              <EuiDescriptionListDescription>{module.type}</EuiDescriptionListDescription>
+            </EuiDescriptionList>
+          </EuiFlexItem>
+
+          {/* COLUMN WITH QUERY */}
+          <EuiFlexItem grow={3}>
+            {module.query !== undefined ? (
+              <EuiFlexGroup direction="column" gutterSize="xl">
+                <EuiFlexItem>
+                  <EuiDescriptionList rowGutterSize="m">
+                    <EuiDescriptionListTitle>
+                      <FormattedMessage
+                        id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.eligibleTitle"
+                        defaultMessage="One or more indices must match the following query:"
+                      />
+                    </EuiDescriptionListTitle>
+                    <EuiSpacer size="m" />
+                    <EuiDescriptionListDescription>
+                      <EuiCodeBlock
+                        language="json"
+                        isCopyable
+                        overflowHeight="500px"
+                        data-test-subj="mlPreconfigJobsQueryBlock"
+                      >
+                        {JSON.stringify(module.query, null, 2)}
+                      </EuiCodeBlock>
+                      <EuiSpacer size="s" />
+                    </EuiDescriptionListDescription>
+                  </EuiDescriptionList>
+                </EuiFlexItem>
+                {module.query !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexEnd">
+                      <EuiFlexItem grow={false}>
+                        <EuiText size={'xs'}>
+                          <FormattedMessage
+                            id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.dataRecognizerHelpButtonLabel"
+                            defaultMessage="Check indices to see which can be used to run jobs in this module?"
```

Comment:
I'd drop the question mark and change `see` for inclusivity. 
```suggestion
                            defaultMessage="Check indices to determine which can be used to run jobs in this module."
```

Reply:
Updated in 9838277f8bb24511c72c3af4da087cbdb52624e7